doc/source/user_guide/beginners_guide.rst

```diff
@@ -0,0 +1,345 @@
+2. FNO Blocks (which include Spectral Layers and MLP Layers)
+3. Output Projection Layer
+
+
+.. image:: /_static/images/fno_marked.jpg
+  :width: 800
+
+Let's dive into each of these components:
+
+1. Data Projection Layer (Lifting)
+----------------------------------
+
+The data projection layer, also known as the lifting layer, is the first step in the FNO architecture. Its purpose is to lift the input data to a higher-dimensional space.
+
+Function:
+
+- Takes the input data and projects it to a higher-dimensional space.
+- Increases the number of channels from the input dimension to the desired hidden dimension.
+
+Implementation: This layer applies a linear transformation to increase the number of channels.
+
+
+.. code-block:: python
+
+    self.P = nn.Linear(self.in_channels, self.hidden_channels)
+
+2. FNO Blocks
+-------------
+
+The core of the FNO architecture consists of multiple FNO blocks. Each block contains two main components:
+
+a) Spectral Layer
+~~~~~~~~~~~~~~~~~
+
+The spectral layer is where the Fourier transform magic happens. It performs convolutions in the Fourier space, which is the key innovation of FNOs.
+
+Function:
+
+- Applies Fourier transform to the input
+- Performs a linear transformation on the lower Fourier modes
+- Applies inverse Fourier transform
+
+Implementation: The spectral layer is implemented using the `SpectralConv2d` class (for 2D problems). Here's a simplified version:
+
+.. code-block:: python
+
+    class SpectralConv2d(nn.Module):
+        def __init__(self, in_channels, out_channels, modes1, modes2):
+            super(SpectralConv2d, self).__init__()
+            self.in_channels = in_channels
+            self.out_channels = out_channels
+            self.modes1 = modes1  # Number of Fourier modes to multiply, at most floor(N/2) + 1
+            self.modes2 = modes2
+            self.scale = (1 / (in_channels * out_channels))
+            self.weights1 = nn.Parameter(self.scale * torch.rand(in_channels, out_channels, self.modes1, self.modes2, dtype=torch.cfloat))
+            self.weights2 = nn.Parameter(self.scale * torch.rand(in_channels, out_channels, self.modes1, self.modes2, dtype=torch.cfloat))
+
+        def forward(self, x):
+            batchsize = x.shape[0]
+            # Compute Fourier coeffcients up to factor of e^(- something constant)
+            x_ft = torch.fft.rfft2(x)
+
+            # Multiply relevant Fourier modes
+            out_ft = torch.zeros(batchsize, self.out_channels, x.size(-2), x.size(-1)//2 + 1, dtype=torch.cfloat, device=x.device)
+            out_ft[:, :, :self.modes1, :self.modes2] = self.compl_mul2d(x_ft[:, :, :self.modes1, :self.modes2], self.weights1)
+            out_ft[:, :, -self.modes1:, :self.modes2] = self.compl_mul2d(x_ft[:, :, -self.modes1:, :self.modes2], self.weights2)
+
+            # Return to physical space
+            x = torch.fft.irfft2(out_ft, s=(x.size(-2), x.size(-1)))
+            return x
+
+b) MLP Layer
```

Comment:
We need to make sure we're clear that the MLP is only applied to channels, see `neuralop.layers.channel_mlp`